pkg/validate/streaming.go

```diff
@@ -18,7 +18,11 @@ package validate
 
 import (
 	"bytes"
+	"fmt"
+	// "io"
 	"net/url"
+	// "os"
```

Comment:
yes, this pr is WIP. will update after we figure out why we can not get the stdout...